moz_security/sandboxes/heka/analysis/moz_security_cloudtrail.lua

```diff
@@ -0,0 +1,147 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+# Mozilla Security Cloudtrail notifications
+
+## Sample Configuration
+```lua
+filename = "moz_security_cloudtrail.lua"
+message_matcher =
```

Comment:
Can we add an example message matcher here, something representative of the actual deployment with the cloudtrail streamer?

Reply:
sorry yes! I left this blank because it was not clear to me what the matcher would look like. Is this something we set in a config file?

Reply:
Something like `Type == 'logging.cloudtrail.lambda.cloudtrail.logs|app.log'` should work

It will depend on what we go with for the stream name